ApplicationConfig/Providers/SiteAccesses.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\ApplicationConfig\Providers;
+
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use eZ\Publish\Core\MVC\Symfony\SiteAccess;
+use EzSystems\PlatformUIBundle\ApplicationConfig\Provider;
+
+class SiteAccesses implements Provider
```

Comment:
@andrerom for now, PlatformUI does not receive the list of siteaccess. Also, so far I don't know what are the plans regarding PlatformUI vs. siteaccesses (cf. the EPIC https://jira.ez.no/browse/EZP-25411)

Instead of a creating a new provider, I would enhance the Language provider to add a `default` key in https://github.com/ezsystems/PlatformUIBundle/blob/master/ApplicationConfig/Providers/Languages.php#L33 to flag the default language. From that, setting the `defaultLanguageCode` app attribute will be trivial.